infra/k8s/sidecar.yaml

```diff
@@ -2,28 +2,35 @@ apiVersion: apps/v1
 kind: DaemonSet
 metadata:
   name: testground-sidecar
+  annotations:
+    cni: "flannel"
 spec:
   selector:
     matchLabels:
       name: testground-sidecar
   template:
     metadata:
+      annotations:
+        cni: "flannel"
       labels:
         name: testground-sidecar
     spec:
       terminationGracePeriodSeconds: 10
       hostPID: true
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
       containers:
       - name: testground-sidecar
-        image: ipfs/testground:latest
+        image: nonsens3/testground:latest
```

Comment:
TODO

Reply:
Will change prior to merging. We are rebuilding the image on `master`, so during development I use my repo.